config/betterquesting/DefaultQuests/Quests/BeeBreeding-AAAAAAAAAAAAAAAAAAAAEw==/UrGhast-AAAAAAAAAAAAAAAAAAABcA==.json

```
@@ -12,7 +12,7 @@
   "properties:10": {
     "betterquesting:10": {
       "autoClaim:1": 0,
-      "desc:8": "Need a lot of magical wood? Well this bee produces carminite.",
+      "desc:8": "Need a lot of magical wood? Well this bee produces carminite.\n\n[note]To get the correct Thaumium block, compress 9 ingots to get the GT Thaumium block and then chisel that block to get the correct type[/note]",
```

This could be made a bit more clear imo.
```suggestion
      "desc:8": "Need a lot of magical wood? Well, this bee produces carminite.\n\n[note]This bee requires a Thaumcraft Thaumium block below the apiary to be bred. To get the correct one, chisel a GT Thaumium block[/note]",
```

---

Yeha it definitely could be, i just took the wording from one of the Thaumcraft Golem Quests, but I can change it